.storybook/preview.tsx

```diff
@@ -543,27 +543,32 @@ const preview: Preview = {
         },
         group1: {
           name: 'Group 1 (240px - 399px)',
-          styles: { width: '399px', height: '900px' },
+          styles: { width: '240px', height: '900px' },
+          type: 'mobile',
+        },
+        group1_320: {
+          name: 'Group 1 - 320px (240px - 399px)',
```

Comment:
this name looks a bit wrong. Think it should be `Group 1 - 320px (320px - 399px)` maybe?